cmake/graph-compiler.cmake

```diff
@@ -32,7 +32,14 @@ if (NOT DEFINED GRAPH_COMPILER_LIBS)
             GcJitWrapper
             GcCpuRuntime
     )
+
+    if (GC_ENABLE_IMEX)
+        add_definitions(-DGC_ENABLE_IMEX)
+        list(APPEND GRAPH_COMPILER_LIBS GcOpenclRuntime)
+    endif()
+
```

Comment:
I'm not sure, whether we should automatically enable GC-GPU capabilities when `-DENABLE_GRAPH_COMPILER=ON` and `-DENABLE_INTEL_GPU=ON` or only if `-DGC_ENABLE_IMEX=ON` was explicitly specified?